models/quantity.rb

```diff
@@ -30,5 +30,11 @@ class Quantity < Base
     # validates_numericality_of :setup_value
 
     belongs_to :unit, class_name: 'Models::Unit', as_json: :id
+
+    def initialize(hash)
+      hash[:empty] = false unless hash.key?(:empty)
+      hash[:fill] = false unless hash.key?(:fill)
```

Comment:
It won't allow to fix the issue of the default value returned with as_json :disappointed: